bee-api/bee-rest-api/CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- ## Unreleased - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security -->
+
+## 0.1.0 - 2021-04-20
+
+### Added
+
+- types module:
+  - `body.rs`
+  - `dtos.rs`
+  - `error.rs`
+  - `responses.rs`
```

Comment:
I wonder how useful that information is. I think it actually makes more sense to mention the introduced types and static methods and if any macros. Imagine reading this a year from now. Then I think it's more interesting to know when we introduced a certain type/static method/macro instead of the module. Wdyt?

Reply:
I agree

Reply:
okey, listing them